pkg/cli/flags_util.go

```diff
@@ -323,6 +323,7 @@ const (
 	tableDisplayRecords
 	tableDisplaySQL
 	tableDisplayHTML
+	tableDisplayRawHTML
```

Comment:
can you add a comment explaining the difference between rawhtml and html?

Reply:
Done.